01/ht/gendos379/03/script.js

```diff
@@ -32,25 +34,27 @@ function enableCaption() {
 };
 
 function tableIsVaild() {
-    if (document.getElementById('rowsNumber').value > 0 && document.getElementById('colNumber').value > 0) {
-        return true
-    }
-    return false
+    return document.getElementById('rowsNumber').value > 0 && document.getElementById('colNumber').value > 0;
+}
+
+function postTableCode() {
+    var tableCodeDiv = document.getElementById('table-code');

```

Comment:
so you
1) clear `.table-code`
2) create new `textarea`
3) add created textarea to clean block

why not just update value for existing textarea?